spec/core/account/schemas/ActivityBucketDTO.yml

```diff
@@ -15,7 +15,7 @@ properties:
     $ref: "../../../schemas/Amount.yml"
     description: Fees paid by the account for this bucket.
   beneficiaryCount:
-    type: integer
+    $ref: "../../../schemas/Uint32.yml"
```

Comment:
Should it be `UInt32` for the naming?

Reply:
Uses the same name than catbuffer for the unit32 type. In open API (java and other languages), the best uint32 representation is a signed int64 type

Reply:
about, capital `I`,  it's ok,  what's your preference @dgarcia360?

Reply:
renamed to UInt32